### PR TITLE
Fix for seeking outside of buffered zone

### PR DESCRIPTION
**Pull request by DonatoB**
[Delivers #91051588]

### CODE REVIEW

File: src/js/providers/html5.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ define([
 ], function(cssUtils, utils, stretchUtils, _, events, states, eventdispatcher, DefaultProvider) {
 
     var clearInterval = window.clearInterval,
+        stallInterval,
+        STALL_DELAY = 100,
+        STALL_TOLERANCE = STALL_DELAY/2,
```
Comment (robwalch) [inline src/js/providers/html5.js]: 50 ms tolerance is too low (an animation lower than 20fps with no audio might not get time updates?). 125 ms (1/8 sec) would be much safer. It's slower than 1/16 -1/10 were the illusion of motion or delays are perceivable, but at least we won't send false stalls.

File: src/js/providers/html5.js [hunk 2]
```diff
@@ -17,16 +20,37 @@ define([
...
+        };
+    };
+
     function _setupListeners(eventsHash, videoTag) {
         utils.foreach(eventsHash, function(evt, evtCallback) {
             videoTag.addEventListener(evt, evtCallback, false);
         });
+
+        var checker = stallCheckGenerator(videoTag, eventsHash.stalled);
+        stallInterval = setInterval(checker, STALL_DELAY);
```
Comment (robwalch) [inline src/js/providers/html5.js]: always call clearInterval(x) before x = setInterval()

File: src/js/providers/html5.js [hunk 3]
```diff
@@ -17,16 +20,37 @@ define([
...
     }
 
     function _removeListeners(eventsHash, videoTag) {
         utils.foreach(eventsHash, function(evt, evtCallback) {
             videoTag.removeEventListener(evt, evtCallback, false);
         });
+
+        if (stallInterval) {
```
Comment (robwalch) [inline src/js/providers/html5.js]: don't need the if. assign a default value of 0 or -1 (pick your fave) to intervalIds so js knows it's a Number

**Comment by DonatoB** [comment]
Will recreate this pull request against master

**Pull request closed without merging**